### ISSUE TITLE
[Bug] Render Issue 

### ISSUE THREAD

**Issue by quickdrcpped** [sp614x/optifine#5187]
## Description of Issue
I'm using schematica and optifine in forge and everytime I open a schematica it does this weird render glitch where bits and pieces of the schematic are all over the place (it's kinda hard to explain should make more sense once you look at the screenshots)
## Steps to Reproduce
1. Put Schematica-1.8.9-1.7.7. and lunatrius core in forge version 11.15.1.1722 with any 1.8.9 optifine version in your mods folder and load up forge

## OptiFine Version
HD U L5 - HD U M5 pre2 (might happen with previous versions before L5 i'm not sure)

## Installation Method
I put the JAR file in my mods folder 

## Fabric/Forge Version
11.15.1.1722

## Other Installed Mods
Crystal Mod 1.8.9 Version 1.4.1 and The 5zig Mod v3.12.5 for minecraft 1.8.9

## Log Files/Crash Reports
https://pastebin.com/ktAbE3HJ

**DO NOT directly copy and paste log contents here on GitHub.**

## F3 Debug Screenshot

To open this menu, simply press `F3` on your keyboard while in-game. Please ensure the text is easily readable. Depending on the size/resolution of your monitor, you may have to adjust your GUI scale for all text to be completely visible on-screen.

## Prior Testing
If you've installed standalone:
- Does this happen in Vanilla without OptiFine? **no**

If you've installed using Forge/Fabric:
- Does this happen without other mods installed? 
   **no**
- Does this happen without any mods at all?
   **no**.
- Does this happen in OptiFine standalone? **no**
- Does this happen in Vanilla without OptiFine? **no**
- Have you tried using a binary search to find which mods cause this issue?
   -Yes it only happens when I'm using schematica and lunatrius core with optifine and also when I use crystal mod with optifine
   - Does the issue still happen with only the remaining mods installed?
 Like I said above only happens while using schematica and lunatrius core with optifine and when I use crystal mod with optifine 

## Additional Information
Here is the link to all the screenshot evidence I have https://imgur.com/a/ea0SdLP

**Comment by MennoGamed** [comment]
This is a simple mistake. The way Schematic works is not possible with Optifine.
I think it was "Render regions" that doesn't work together. Not 100% sure about this.

In any case, there is 1 setting you should turn off. That should fix the problem.

**Comment by quickdrcpped** [comment]
> This is a simple mistake. The way Schematic works is not possible with Optifine.
> I think it was "Render regions" that doesn't work together. Not 100% sure about this.
> 
> In any case, there is 1 setting you should turn off. That should fix the problem.

Yeah I actually figured this out by playing with my settings a few days after I posted this bug report. (it was render regions causing the graphics with schematica to bug out)

**Comment by MennoGamed** [comment]
Great to hear that you already found it yourself.
Anyway, it works!
